receiver/k8sobjectsreceiver/config.go

```diff
@@ -33,6 +34,14 @@ var modeMap = map[mode]bool{
 	WatchMode: true,
 }
 
+type ErrorMode string
+
+const (
+	PropagateError ErrorMode = "propagate"
```

Comment:
I also don't understand what Propagate means here - are we propagating errors from somewhere?